apiary.apib

```diff
@@ -954,11 +954,570 @@ Outputs the list of peers that were removed.
 
 # Group config
 
-## edit
+## config [POST /config{?arg1,arg2}{&bool,json}]
```

Comment:
in fact, go-ipfs HTTP API doesn't look at the method, so you can call it with any.